packages/core/src/utils.ts

```diff
@@ -270,11 +270,6 @@ export function getQuestionnaireAnswers(
   return result;
 }
 
-/**
- * Recursively builds the questionnaire answer items map.
- * @param items The current questionnaire response items.
- * @param result The cumulative result map.
- */
 function buildQuestionnaireAnswerItems(
```

Comment:
QuestionnaireForms.tsx is still using this method for now until we have the isQuestionEnabled set for getAllQuestionnaireAnswers

Reply:
Can you elaborate on this?  Is that something we need to fix?  It looks like the new methods are almost exact duplicates of the existing methods, so I'm inclined to just go ahead and fix whatever we need to fix in `isQuestionEnabled`